src/summaries/introduction_to_defi.yaml

```diff
@@ -0,0 +1,100 @@
+      * Banks do not allow micro-transitions.
+      * It is difficult to get paid using banks.
+    
+    - Lack of Interoperability
+      * Institutions are isolated from each other and have no interlinking.
+      * It is difficult to move funds from one bank to another.
+      * Difficult to move money from a bank to non bank.
+      
+    - Opacity
+      * Banking institutions have very little transparency.
+      * Bank customers do not know the health of the bank.
+      * Individuals are forced to rely on costly regulation and promise of bailouts.
+
+    These problems when not tackled lead to following destructive effects.
+
+    - Missed Growth Opportunities
+      * Entrepreneurs who have suppose 25% ROI on their great idea, but their project is not executed due to high cost of credit card borrowing.
+      * High ROI project fuels high CAPEX, strong real GDP growth, and robust employment growth.
+
+    - Inequality of opportunities
+      * Projects should be financed based on the quality of the idea and the soundness of the execution plan.
+      * Many have no access to internet commerce (to buy or sell).
+      * Given the number of unbanked and underbanked, this creates unequal opportunities and perpetuates or even exacerbates inequality.
+
+
+
+    
+- title : Decentralised Finance Usage 
+  shortTitle: Uses of DeFi
+  key : uses-of-decentralised-finance
+  details: |
+    Centralised Finance opens up a wide range of advancement in the current centralised system.
+
+    - Transfer for Money 
+      * Using DeFi sending money would be as easy as sending an email.
+      * You just need to enter other persons account address and your payment will directly go into their account.
+
+    - Stream money around the globe
+      * One can stream money over ethereum
+      * This lets you pay someone their salary by the second, giving them access to their money whenever they need it. Or rent something by the second like a storage locker or electric scooter.
+
+    - Access to stablecoins
+      * Volatility in the cryptocurrency is of huge concern which hinders its growth as well.
+      * DeFi community has introduced stablecoins.
+      * Their value remains peggged to another asset, usually a popular currency like dollar.
+      * Coins like Dai or USDC have a value that stays within a few cents of a dollar.
+
+    - Borrowing 
+      * DeFi helps in borrowing money with privacy. 
+      * The borrower must put up collateral that the lender will automatically receive if their loan is not repaid. 
+      * This allows you to borrow money without credit checks or handing over private information.
+
+    - Flash loans
+      * DeFi introduced the concept of flash loans where the borrower gets loans without handling any collateral to lender.
+      * If the borrower is not able to pay the loan, the transaction is reversed as if it never occured.
+
+    - Tokenization
```

Comment:
Other Benefits
- Programmable Incentives
- Composability (Building on top of other protocol)
- Easy to build applications/products (Cloud simplified creation of applications. Blockchain simplifies creation of financial apps)
- Its global